components/user_settings/display/user_settings_theme/user_settings_theme.test.jsx

```diff
@@ -43,5 +44,26 @@ describe('components/user_settings/display/user_settings_theme/user_settings_the
 
         expect(requiredProps.updateSection).toHaveBeenCalledTimes(1);
         expect(requiredProps.updateSection).toHaveBeenCalledWith('');
+
+        expect(requiredProps.actions.saveTheme).toHaveBeenCalled();
+    });
+
+    it('should deleteTeamSpecificThemes if applyToAllTeams is enabled', async () => {
+        const props = {
+            ...requiredProps,
+            actions: {
+                saveTheme: jest.fn(() => Promise.resolve()),
+                deleteTeamSpecificThemes: jest.fn(() => Promise.resolve()),
```

Comment:
Please resolve with expected value (or can be empty object) instead of null, to prevent `UnhandledPromiseRejectionWarning`.  Same case for others (only those related to this PR).